composer.json

```diff
@@ -8,14 +8,14 @@
             "email": "greg.1.anderson@greenknowe.org"
         }
     ],
-    "autoload":{
-        "psr-4":{
+    "autoload": {
+        "psr-4": {
             "Consolidation\\OutputFormatters\\": "src"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Consolidation\\TestUtils\\": "tests/src"
+            "Consolidation\\OutputFormatters\\Tests\\": "tests/src"
```

Comment:
Probably shouldn't change the namespace here; these are not the OutputFormatters tests; they are utility classes used by the tests. I would rather not conflate the two.